### ISSUE TITLE
[Remote rendering problem] Running OpenXR on desktop

### ISSUE THREAD

**Issue by whatthemehek** [Azure/azure-remote-rendering#109]
I'm trying to run ARR as an OpenXR app on desktop, but currently when I enable OpenXR for desktop in Project Settings, the remote model stops displaying. If I disable that setting, it displays correctly. Is there a way to create an OpenXR app for desktop? Can I switch the graphics bindings or something?

**Comment by whatthemehek** [comment]
Additionally, even if I don't enable OpenXR for PC Standalone, when following the instructions to run the default ARR build on UWP for PC, even though it works fine on device and on headset, when I build UWP as a release build for PC on Unity 2020.3, x86, and try to run the .sln through Visual Studio 2022, I just get a white screen app. Is there any known cause for this?

**Comment by ChristopherManthei** [comment]
Hi @whatthemehek ,
On desktop, VR is currently not a supported platform, please make a feature request at our [Feedback Page](https://feedback.azure.com/d365community/forum/46aa4cc0-fd24-ec11-b6e6-000d3a4f07b8).
Sorry the [system requirements](https://learn.microsoft.com/azure/remote-rendering/overview/system-requirements) page isn't clear enough on the supported graphics binding. We will update our page accordingly.

You can still use desktop simulation on both UWP desktop and Windows Standalone. For it to work, you need to disable Unity **XR Initialize of Startup** or disable all XR plugins.
<img width="374" alt="image" src="https://github.com/Azure/azure-remote-rendering/assets/22638947/697fcc14-18dc-43ce-b739-d245a9a5b608">

**Comment by whatthemehek** [comment]
Hey- just curious, what is the stopgap on supporting OpenXR on desktop? What is causing the display error?

**Comment by ChristopherManthei** [comment]
Hi @whatthemehek , so far there wasn't any customer signal so we didn't invest time into it.

**Comment by whatthemehek** [comment]
I see! Consider this a customer signal :D. Is there any way programmatically I personally would be able to hack ARR for desktop OpenXR on my device- changing some code flags or something?

**Comment by ChristopherManthei** [comment]
Hi @whatthemehek,
While OpenXR is cross platform, there are many device specific extensions that are necessary to provide the best experience for each device. So far, we have focused on HoloLens2 and Quest 2 / Pro.
Thank you for your feedback, we have created a ticket in our backlog to investigate what is necessary to get OpenXR running on desktop. Right now, there is unfortunately no way to get this working on your own.
